README.md

```diff
@@ -63,6 +66,7 @@ CSS
 }
 
 ```
+- `IGNORE INLINE STYLE` rule will not add any inline styles into matched selector's. (e.g. --darkreader-inline-bgcolor)
```

Comment:
Imagine you don't know how Dark Reader works. Will you understand what `IGNORE INLINE STYLE` does after reading this? It should say, that it makes Dark Reader ignore inline styles for the element. Let's also move it after `INVERT` and `CSS` descriptions.